governator-servlet/src/main/java/com/netflix/governator/guice/servlet/GovernatorServletContextListener.java

```
@@ -85,12 +99,15 @@ public void contextDestroyed(ServletContextEvent servletContextEvent) {
      */
     protected final Injector getInjector() {
         try {
-            return createInjector();
+            if (injector == null) {
```

We should throw an exception if injector != null